src/theme.ts

```diff
@@ -243,6 +302,200 @@ const theme = createTheme({
   },
 });
 
+//The new theme palette
+const newThemePalette = {
+  activityStatusColors: {
+    cancelled: '#6D6D6D',
+    draft: '#F3C81C',
+    expired: '#D1D1D1',
+    published: '#1B9E4B',
+    scheduled: '#3E6CD2',
```

Comment:
I have concerns with hardcoding colors like this. Let me try to explain.

I think of theme colors as falling into one of two categories.

Some colors are **color primitives**, i.e. concrete colors such as `#BED0F9` from the new design system color palette. There are 60 or so colors like these and I'm not sure what is the best way to name them, but @theo-organism has currently named them "Blue 1" etc and I think that's good.

![image](https://github.com/user-attachments/assets/b2425494-f8d9-4ff2-9421-700d0b72b102)

The other category of colors are **use case colors**, such as "primary button color", "active chip color" etc. These all reference colors from the big color palette, and I think it would make sense to have them as references (i.e. variables) in the code as well.

So I believe there should only be one section of the theme that contains "hard-coded" hex values, and that's the big "primitive" color palette. All other parts of the theme should reference the colors from that first part. That way, each hex value only exists in one place.

Reply:
I agree! My process with this is that I started with just "doing", to make it bad first, and then good. I think implementing the colours in the theme is a separate PR when the person writing that can focus only on making the colour system good. So, my reasoning is that it is messy now because it will be made better later :)

Reply:
Just a small correction in terms of how the colors are used. See the Figma page below:
https://shorturl.at/RknG5

The term I have been using that Richard is referring to as "color primitives" is called just "Palette" in the Figma library right now and colors that have been assigned a specific purpose are part of the "Contexualized palette". These terms are not necessarily final, and I do like the term color primitives, but that is what I have called them until now. 

However, what I wanted to clarify was that the colors used for the states e.g. "published", "draft", etc, are using colors not from the Chip color collection, which Richard referred to above, but are instead using colors from the Red, Green, Blue and Yellow (RGBY) only palettes further to the left in the document. Those colors are taken from Tailwind and are called for example Green-100, Green-200 and so on.

See image below:

<img width="1261" alt="image" src="https://github.com/user-attachments/assets/db423731-6c5b-44dd-a39c-29103a3941b3">

The Chip colors are however derived from the main colors of RGBY-only palettes.